safenode/src/client/chunks/mod.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+mod error;
+mod pac_man;
+
+pub(crate) use self::error::{Error, Result};
+pub(crate) use pac_man::{encrypt_large, to_chunk, DataMapLevel};
+
+use bytes::Bytes;
+use self_encryption::MIN_ENCRYPTABLE_BYTES;
```

Comment:
I was wondering if we could get rid of the SmallFile/BigFile distinction, and just padd smaller files with `0`s until we can encrypt them? How much is `MIN_ENCRYPTABLE_BYTES`? If it's small enough then maybe it's reasonable?
This would get rid of a lot of complexity too. This always bugged me somehow in sn.

Reply:
`MIN_ENCRYPTABLE_BYTES` is 3072. since min size of a chunk is 1024 and it requires at least 3 to be able to do self-encryption.

Yeah, we could perhaps pad smaller files. A question we can bring up right away in slack.

Reply:
Why don't we make 4096 size chunks? And fix this for good.